modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -27,21 +27,62 @@ just like your other ownCloud files and folders.
 
 Compared to standard SMB access, WND has advanced features like:
 
-* User lockout prevention and password reset
-* More authentication mechanisms against the backend
-* Change notifications
-* Enhanced ACL support
+. User lockout prevention and password reset
+. More authentication mechanisms against the backend
+. Listen to change information triggered by the backend
+. Enhanced ACL support
+. Collaborative WND (CWND)
 
-Here is a brief description of the advanced features:
+.Brief Description of Advanced Features:
 
+User lockout prevention and password reset::
 Depending on the Windows or Samba policy, the user could get locked out of his account if he enters a wrong password a number of times. The lockout prevention tries to avoid this from happening, by resetting the password if it is wrong. For the case of ownCloud's standard SMB connector, the password won't be reset. It could happen, that the user gets locked out of the file server.
 
+More authentication mechanisms against the backend::
 Please see the details about the xref:enterprise/external_storage/enterprise_only_auth.adoc[Enterprise-Only Authentication Options]
 
-WND has the ability to trigger change notifications to users based on changes for files and folders. 
+Listen to change information triggered by the backend::
+Native Windows File Servers provide the ability to proactively send change notifications made to files and folders somewhere in a share. This mechanism then updates the ownCloud database and provides the changes made to accessing users. Users do not need to manually check for changes in all possible locations of their mount. Changes processed are also propagated to sync clients automatically.
 
-With enhanced ACL support, both SMB and WND evaluate the file attributes (whether the file is hidden or read-only) to decide what ownCloud permissions should the file or folder have in ownCloud. On top of this, WND can also evaluate the ACLs by using the "ocLdapPermissionManager" in the mount point configuration. This will bring more accurate permissions to ownCloud, specially if each user could have different permissions over the files in Windows.
+Enhanced ACL support::
+With enhanced ACL support, both SMB and WND evaluate the file attributes (whether the file is hidden or read-only) to decide what ownCloud permissions should the file or folder have in ownCloud. On top of this, WND can also evaluate the ACLs by using the `ocLdapPermissionManager` in the mount point configuration. This will bring more accurate permissions to ownCloud, specially if each user could have different permissions over the files in Windows.
```

Comment:
It might be important to warn that this ACL support doesn't apply to the collaborative WND (at least for now)

Reply:
👍

Reply:
Can you tell more about. Is it just not working? or ist it breaking something? What is the effect the user / admin is confronted with?

Reply:
For listing, we're using the service account, which could have specific permissions. Since we're using the SMB file attributes by default, unless the file is read-only, it will have all permission. Those permissions are stored in the filecache.

Problem 1: if we evaluate ACLs for the service account, the ownCloud permissions will be limited to read-only (unless the service account have permissions to write or full permissions in the SMB server).

Problem 2: the permissions are cached, but they aren't re-evaluated unless the file changes. For the same fileid, each user could have different permissions. If userA has only read access, we might cache that the permissions for the fileid are read-only. However, when userB access, since the permissions are cache, he'll also have read-only access (in ownCloud) even though he could have full access in the SMB server.

For now, ownCloud is being permissive by evaluating only the file attribute, so usually, ownCloud will allow anything to any user. In any case, the permissions are being enforced by the SMB server, so if userA cannot write a file an error will be reported.

Reply:
As discussed, I added a text at the end of this description to clarify the mandatory use of the default nullperm manager when using cwnd.